build.sbt

```diff
@@ -11,7 +11,9 @@ resolvers ++= {
   else Seq.empty
 }
 
-libraryDependencies += "org.scoverage" %% "scalac-scoverage-plugin" % "1.2.0-SNAPSHOT"
+libraryDependencies += "org.scoverage" %% "scalac-scoverage-plugin" % "1.2.0"
+
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.12")
```

Comment:
I really don't think it is good idea to force users of the sbt-scoverage plugin to download sbt-scalajs, I see all you need from that plugin is a way to guess if the project is Scala.JS and what "binary" version it uses. I guess sbt should have such feature, but for now I think it makes much more sense to just guess that as I did: https://github.com/scoverage/sbt-scoverage/pull/166/commits/2c1e7bb979a7a9720841a6222c11e0f18799c65a#diff-627f9fd5c400939149d75695caad1988R68